manifests/init.pp

```diff
@@ -225,7 +225,7 @@
   Hash $nginx_upstreams                                   = {},
   Nginx::UpstreamDefaults $nginx_upstreams_defaults       = {},
   Boolean $purge_passenger_repo                           = true,
-  String[1] $nginx_version                                = pick(fact('nginx_version'), '1.6.0'),
+  String[1] $nginx_version                                = pick(fact('nginx_version'), '1.20.0'),
```

Comment:
This looks like an unrelated change. Is there something so this shows up in the changelog?

Reply:
removed it